algorithms/ar-isrt/javascript/insertion_sort.js

```diff
@@ -0,0 +1,20 @@
+let insertionSort = (array) => {
+  for (outer = 1; outer < array.length; outer++) {
```

Comment:
Codacy found an issue: ['outer' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6335885)